cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -737,23 +737,23 @@
   <!-- Kafka Server Configuration -->
 
   <property>
-    <name>kafka.bind.address</name>
+    <name>kafka.server.bind.address</name>
```

Comment:
Rather than just changing these (which deletes them from the documentation!), these should be duplicated, marked as deprecated, with a note about the new versions to be used. They can be deleted in a later version of CDAP.